wurst/systems/spells/gatherer/TeleGathering.wurst

```diff
@@ -15,17 +15,44 @@ let ABILITY_ID_OMNI_TELEGATHER = 'A06V'
 let ABILITY_ID_CASTER_BUFF = 'A056'
 let CASTER_BUFF_ID = 'B00H'
 let TARGET_BUFF_ID = 'B00I'
-let SORT_DISTANCE_INCREMENT = 150
+let SORT_DISTANCE_INCREMENT = 110
+let SORT_DISTANCE_BASE = 30
```

Comment:
How does add a base distance play with the range you look for existing items? You probably need to adjust that.

Reply:
Good point, fixing